x-pack/plugins/fleet/common/types/models/agent_policy.ts

```diff
@@ -35,7 +35,7 @@ export interface NewAgentPolicy {
 export interface AgentPolicy extends Omit<NewAgentPolicy, 'id'> {
   id: string;
   status: ValueOf<AgentPolicyStatus>;
-  package_policies: string[] | PackagePolicy[];
+  package_policies?: PackagePolicy[];
```

Comment:
cool. I like that you still left this property around, since it can be useful sometimes to have it all in one API call.